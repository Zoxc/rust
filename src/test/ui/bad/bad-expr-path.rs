mod m1 {}

fn main(arguments: Vec<String>) {
    log(debug, m1::arguments);
    //~^ ERROR cannot find function `log` in this scope
    //~| ERROR cannot find value `debug` in this scope
    //~| ERROR cannot find value `arguments` in module `m1`
}
