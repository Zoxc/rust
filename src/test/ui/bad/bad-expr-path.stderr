error[E0425]: cannot find function `log` in this scope
  --> $DIR/bad-expr-path.rs:4:5
   |
LL |     log(debug, m1::arguments);
   |     ^^^ not found in this scope

error[E0425]: cannot find value `debug` in this scope
  --> $DIR/bad-expr-path.rs:4:9
   |
LL |     log(debug, m1::arguments);
   |         ^^^^^ not found in this scope

error[E0425]: cannot find value `arguments` in module `m1`
  --> $DIR/bad-expr-path.rs:4:20
   |
LL |     log(debug, m1::arguments);
   |                    ^^^^^^^^^ not found in `m1`

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0425`.
